src/styles/nav.scss

```
@@ -0,0 +1,76 @@
+@import "./variables";
+
+.nav {
+  width: max-content;
+  display: block;
+  height: 60px;
+  cursor: pointer;
+
+  a {
```

```suggestion
  &__link {
```
стилізуй тільки по класам, у тегів низька специфічність і це не по БЕМ